### ISSUE TITLE
package.json in the new ui

### ISSUE THREAD

**Issue by ioggstream** [italia/api-oas-checker#95]
## I expect

no warnings when running the new ui

## Instead

```
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```

**Comment by emilioSp** [comment]
This is due to the temporary presence of npm and yarn as a package manager in the branch.
I suggest using yarn for these [reasons](https://github.com/italia/form-pa/issues/28).
Currently, the branch is in the middle of a migration from npm to yarn. We need to replace yarn with npm in the build process.

**Comment by ioggstream** [comment]
Pick your favourite tool and remove the other :) Go Go Go :DDD

**Comment by ioggstream** [comment]
We can close bugs in new-ui
